README.md

```diff
@@ -237,6 +264,12 @@ The Open/Close trait is used for devices that can be opened and closed such as d
         - Open Command: Only available if Query Only Open/Close is unset.  A device command used to open the device.  Should not require any parameters.  Maps to `open` by default.
         - Close Command: Only available if Query Only Open/Close is unset.  A device command used to close the device.  Should not require any parameters.  Maps to `close` by default.
 
+### Reboot
+
+The Reboot trait is used for devices that can be rebooted.  It can be controlled by saying things like "Hey Google, reboot {device}".  It has the following configuration parameters:
+
+- Reboot Command: A device command to reboot the device.  Maps to `reboot` by default.
```

Comment:
IMO this should have no default value since there's no matching Hubitat capability.

Reply:
Removed non-matching defaults.